app/src/main/java/org/projectbuendia/client/ui/OdkActivityLauncher.java

```diff
@@ -102,6 +132,11 @@ public static void fetchAndShowXform(
         final int requestCode,
         @Nullable final org.odk.collect.android.model.Patient patient,
         @Nullable final Preset fields) {
+        LOG.i("Trying to fetch it from cache.");
```

Comment:
Sorry about that, @dancunningham. I completely forgot to update the PR description. 
Initially, it was fetching from the server first, but after discussing with @capnfabs, we've realized that this was not necessary because now the form are synched frequently, together with other datas, and a profile change is much less frequent.  So, i agree that the apps should priorize the cached form. To invalidade the cache, you just need to go to settings and invalidade all the other datas. This is actually the big point. The forms work as the other datas. You don't need to check anywhere that you are working on offline mode.